package.json

```diff
@@ -17,17 +17,30 @@
   "scripts": {
     "setup": "cd open-simulation-interface && pip install ./ && cd .. && rm -rf open-simulation-interface/osi3 proto",
     "generate": "buf generate",
-    "build": "npm run setup && npm run generate && tsup index.ts --format cjs,esm --dts",
+    "build": "yarn run setup && yarn run generate && tsup index.ts --format cjs,esm --dts",
+    "lint:ci": "eslint --report-unused-disable-directives .",
+    "lint": "eslint --report-unused-disable-directives --fix .",
     "test": "echo \"Error: no test specified\" && exit 1"
   },
   "main": "dist/index.js",
   "module": "dist/index.mjs",
   "types": "dist/index.d.ts",
+  "files": [
+    "dist",
+    "README.md"
+  ],
+  "publishConfig": {
+    "access": "public"
+  },
   "devDependencies": {
     "@bufbuild/buf": "^1.45.0",
     "@bufbuild/protobuf": "^2.2.0",
     "@bufbuild/protoc-gen-es": "^2.2.0",
+    "@eslint/js": "^9.16.0",
+    "eslint": "^9.16.0",
```

Comment:
Question: is that here a specific config for eslint that we shall use across all repos in this organization? https://github.com/Lichtblick-Suite/eslint-plugin

Reply:
Linting now follows the lichtblick standards, one thing to note however, @typescript-eslint does not support typescript v5.7.2, it only supports versions >=v4.3.5 <v5.3.0.

Everything seems to work fine, but we get the following warning when linting:

```
WARNING: You are currently running a version of TypeScript which is not officially supported by @typescript-eslint/typescript-estree.

You may find that it works just fine, or you may not.

SUPPORTED TYPESCRIPT VERSIONS: >=4.3.5 <5.3.0

YOUR TYPESCRIPT VERSION: 5.7.2

Please only submit bug reports when using the officially supported version.
```

Reply:
Should we just use 5.3 in this repo? Or accept the warning?